src/vs/platform/extensionManagement/node/extensionGalleryService.ts

```diff
@@ -123,6 +123,8 @@ const PropertyType = {
 	Dependency: 'Microsoft.VisualStudio.Code.ExtensionDependencies',
 	ExtensionPack: 'Microsoft.VisualStudio.Code.ExtensionPack',
 	Engine: 'Microsoft.VisualStudio.Code.Engine',
```

Comment:
Same here, does it make more sense to use `VscodeEngine` and `AzdataEngine`

Reply:
same as above, will keep the engine and rename to AzDataEngine